### ISSUE TITLE
Remove deleted versions from Metacat Coordinating nodes

### ISSUE THREAD

**Issue by elopatin-uc3** [CDLUC3/mrt-doc#37]
When we updated the resource maps for the UCD ICE and DataUP collections, we did not completely delete the old versions in Metacat. Metacat still points to 31,446 objects in the Merritt repository, when there are only 16,750 objects in the ICE collection. 
 - [ ]

**Comment by elopatin-uc3** [comment]
Mark has been working with Chris Jones to fix this. (Perry Willett - Nov 1, 2013)

**Comment by elopatin-uc3** [comment]
DataONE developers have fixed this in their dev environment, and new version should be released in production the week of Nov 12. (Perry Willett - Nov 8, 2013)

**Comment by elopatin-uc3** [comment]
We're waiting on DataONE to upgrade Coordinating Node software. The upgrade has been postponed; no schedule for completion given.  (Perry Willett - Nov 26, 2013)

**Comment by terrywbrady** [comment]
@elopatin-uc3 , this should be covered by the storage scan effort ... or this will still be a dataone exception which is likely tracked in a new ticket

**Comment by elopatin-uc3** [comment]
Closing as I'd rather track new work in a new ticket (with known, well-defined tasks).
